tests/images/create.js

```diff
@@ -0,0 +1,24 @@
+exports.default = [
+	'.gitignore',
+	'package.json',
+	'README.md',
```

Comment:
Even when we use `node-glob`, it sorts in the alphabetical order AFAIK. This might be an issue.

Reply:
It does. The above _is_ alpha-sorted 😄

Reply:
and it always fails in my machine 😛